UQL/Interpreter.php

```diff
@@ -277,14 +289,24 @@ private function parseValue($value)
      * @param ASTFunctionCall $functionCall
      *
      * @return mixed
-     * @throws Exception\UQLInterpreterException
+     * @throws FunctionNotFoundException
+     * @throws UQLInterpreterException
      */
     private function callFunction(ASTFunctionCall $functionCall)
     {
+        $functionName = $functionCall->getFunctionName();
```

Comment:
We could wrap this new block into private method, so it's easier to understand what is does.